runelite-client/src/main/java/net/runelite/client/plugins/stretchedfixedmode/StretchedFixedModePlugin.java

```diff
@@ -83,6 +85,18 @@ protected void shutDown() throws Exception
 		mouseManager.unregisterMouseWheelListener(mouseWheelListener);
 	}
 
+	@Subscribe
+	public void onResizableChanged(ResizeableChanged event)
```

Comment:
the correct method name should be onResizeableChanged not onResizableChanged

Reply:
Resizable is the correct spelling in English

Reply:
Doesn't matter, the event is named ResizeableChanged not ResizableChanged

Reply:
The name is still wrong